.changeset/polite-ghosts-call.md

```diff
@@ -0,0 +1,6 @@
+---
+'@hashicorp/platform-markdown-utils': minor
```

Comment:
Should this be a `major` as we're now planning to expose native ESM?

Reply:
We're still on the 0.x.x series so this is technically a major.